lib/gctime.rb

```diff
@@ -1,44 +1,53 @@
 # frozen_string_literal: true
-
 require_relative "gctime/version"
 
 module GCTime
-  @mutex = Mutex.new
-  @total_time = 0.0
-
-  begin
-    GC.stat(:time) # Implemented on JRuby and TruffleRuby
-    class << self
-      def enable
-      end
+  if GC.respond_to?(:total_time)
+    # We're on Ruby 3.1+, nothing to backport.
+  elsif GC.stat.key?(:time)
+    # We're likely on JRuby or TruffleRuby
+    module Backport
+      def measure_total_time
+        true
+      end unless GC.respond_to?(:measure_total_time)
 
-      def disable
-      end
+      def measure_total_time=(enabled)
+        # noop
+      end unless GC.respond_to?(:measure_total_time=)
 
       def total_time
-        GC.stat(:time).to_f
-      end
+        (stat(:time) * 1_000_000.0).to_i # nanoseconds integer
```

Comment:
If `GC.stat(:time)` is always an Integer (seems so), then this can be just `stat(:time) * 1_000_000`, right?
Not sure what this `stat()` call end up calling though.

Reply:
Fixed in #2